lib/insights_cloud/async/cloud_connector_announce_task.rb

```diff
@@ -0,0 +1,44 @@
+module InsightsCloud
+  module Async
+    class CloudConnectorAnnounceTask < ::Actions::EntryAction
+      def self.subscribe
+        Actions::RemoteExecution::RunHostsJob
+      end
+
+      def self.connector_feature_id
+        @connector_feature_id ||= RemoteExecutionFeature.feature!(ForemanRhCloud::CloudConnector::CLOUD_CONNECTOR_FEATURE).id
```

Comment:
It would be an error state, since I add this feature in the [engine](https://github.com/theforeman/foreman_rh_cloud/blob/65660aaa793d8c64d331095d7b825db06bfd6d3b/lib/foreman_rh_cloud/engine.rb#L163)